### DISCUSSION TITLE
Merge queue: success PR not being merged

### DISCUSSION THREAD

**Discussion by csantos1113** [community/community#18497] [Pull Requests]
Had 4 PRs in the queue, first one got merged, second one got ✅ required checks, green light to be merged, but it is just seating there.

<img width="649" alt="image" src="https://user-images.githubusercontent.com/9648559/173384454-b4e3e639-cb63-4ef8-968e-0c4b388de8a1.png">

This just started happening today, without making any changes to our merge-queue config.

UPDATE: all 3 PRs in the queue got ✅ required checks, but all of them are stuck in the queue. For some reason they aren't merged

**Comment by dhess** [comment]
This has just now happened to us, as well. We have 3 PRs marked  ✅ stuck in the queue for several minutes now. The builds did completely successfully and there are no more builds running.

**Comment by alexanderniebuhr** [comment]
Also, have this somehow regularly that the queue gets stuck. If we remove the first one, the rest of the PRs finish fine.
